src/components/terminal/DockerTerminal.tsx

```diff
@@ -138,35 +130,119 @@ interface RouteParams {
   name: string;
 }
 
+enum FontSizeChangeType {
+  INCREASE = 'INCREASE',
+  DECREASE = 'DECREASE',
+  RESET = 'RESET',
+}
+
+const FONT_SIZE_DEFAULT = 12;
+const FONT_SIZE_MIN = 10;
+const FONT_SIZE_MAX = 20;
+
+const termOptions: ITerminalOptions = {
+  fontFamily: "source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace",
+  fontSize: FONT_SIZE_DEFAULT,
+  lineHeight: 1.2,
+  cursorBlink: true,
+  cursorStyle: 'bar',
+  allowTransparency: true,
+  theme: nord,
+};
+
 const DockerTerminal: React.FC = () => {
   const { l } = usePrefixedTranslation('cmps.terminal.DockerTerminal');
   const { notify } = useStoreActions(s => s.app);
   const { type, name } = useParams<RouteParams>();
   const termEl = useRef<HTMLDivElement>(null);
+  const terminal = useRef<Terminal>();
+
+  const changeFontSize = (changeType: FontSizeChangeType) => {
+    if (
+      terminal.current &&
+      terminal.current.options &&
+      terminal.current.options.fontSize
+    ) {
```

Comment:
You can use `if (terminal?.current?.options.fontSize)` instead to simplify the condition.

Reply:
Of course. Somehow it was showing an error for me earlier when using optional chaining. Fixed now.